backends/edk2-compat/edk2-svc-write.c

```diff
@@ -5,127 +5,135 @@
 #include <stdio.h>
 #include <errno.h>
 #include <string.h>
-#include <stdlib.h>// for exit
+#include <stdlib.h> // for exit
 #include <argp.h>
 #include "secvarctl.h"
-#include "backends/edk2-compat/include/edk2-svc.h"// import last!!
+#include "backends/edk2-compat/include/edk2-svc.h" // import last!!
 
-
-static int updateSecVar(const char *var, const char *authFile, const char *path, int force);
+static int updateSecVar(const char *var, const char *authFile, const char *path,
+			int force);
```

Comment:
Seeing this around a bit, I'm assuming the misalignment here is coming from github doing weird things to the format? Might need to tweak that.

Also maybe we should consider dropping, or extending the 80-character width limit, since that's a bit much

Reply:
I checked it out locally (`git fetch origin pull/13/head:cf`) and it looks fine to me. 3 tabs, each 8 spaces wide, is 24 characters, and `static int updateSecVar(` is also 24 characters.